clients/python/agentic-sandbox-client/k8s_agent_sandbox/async_connector.py

```
@@ -270,10 +304,231 @@ async def send_request(self, method: str, endpoint: str, **kwargs) -> httpx.Resp
+            import grpc
+        except ImportError as e:
+            raise ImportError(
+                "the sandboxd runtime requires gRPC support; install the "
+                "'grpc' extra: pip install k8s-agent-sandbox[grpc]"
+            ) from e
+
+        async with self._lifecycle_lock:
+            self._ensure_open()
+            async with self._grpc_lock:
+                if not self.grpc_target:
+                    base_url, self.grpc_target = await self._sandboxd_strategy.connect()
```

Ruff (0.16.2)

[warning] Unpacked variable `base_url` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)